src/app/Instance/DeleteInstance/DeleteInstance.tsx

```diff
@@ -54,15 +54,22 @@ const DeleteInstance = (props: DeleteInstanceProps): JSX.Element => {
       setPreloading(false);
       if (processorListResponse.total && processorListResponse.total > 0) {
         setDeleteBlockedReason(
-          t("instance.errors.cantDeleteBecauseProcessorsInside")
+          <Trans
```

Comment:
Out of interest, why do we need `<Trans..>` here where use of `t(..)` works elsewhere?

Reply:
Mainly because in the message we need to translate, there is a `<strong>` tag that we need to render via the `<Trans>` component.